front/src/routes/integration/all/nuki/index.js

```diff
@@ -0,0 +1,41 @@
+import { Component } from 'preact';
+import { connect } from 'unistore/preact';
+import actions from './actions';
+import NukiPage from './NukiPage';
+import DeviceTab from './device-page/DeviceTab';
+import { WEBSOCKET_MESSAGE_TYPES } from '../../../../../../server/utils/constants';
+
+class NukiIntegration extends Component {
+  componentWillMount() {
+    this.props.getIntegrationByName('nuki');
+    this.props.getNukiDevices();
+    this.props.getHouses();
+    this.props.loadProps();
+    this.props.session.dispatcher.addListener(
+      WEBSOCKET_MESSAGE_TYPES.NUKI.CONNECTED,
+      this.props.displayConnectedMessage
+    );
+    this.props.session.dispatcher.addListener(WEBSOCKET_MESSAGE_TYPES.NUKI.ERROR, this.props.displayNukiError);
+  }
+
+  componentWillUnmount() {
+    this.props.session.dispatcher.removeListener(
+      WEBSOCKET_MESSAGE_TYPES.NUKI.CONNECTED,
+      this.props.displayConnectedMessage
+    );
+    this.props.session.dispatcher.removeListener(WEBSOCKET_MESSAGE_TYPES.NUKI.ERROR, this.props.displayNukiError);
+  }
+
+  render(props, {}) {
+    return (
+      <NukiPage>
+        <DeviceTab {...props} />
```

Comment:
On this page, I get no message if the service is not configured

The Zigbee2mqtt integration is a good example: 


![Screenshot 2025-05-08 at 09 18 09](https://github.com/user-attachments/assets/dc267df0-b574-4085-b286-95bc9d3de4d6)